openedx/core/djangoapps/profile_images/tests/test_images.py

```diff
@@ -23,6 +23,7 @@
     validate_uploaded_image,
     _get_exif_orientation,
     _get_valid_file_types,
+    _update_exif_orientation,
```

Comment:
Heh, this is technically correct, but this is one of the few times where we don't want to make this kind of change. In this case, this is a line added by edX. While we would prefer they always include the comma, sadly, they don't. Even though including it is the "right thing" to do, we want to avoid diverging from edX. In this case, us adding this when edX doesn't, means there's an increased chance that we end up with a merge conflict on this line, the next time we pull code from edx.
As such, let's revert this change.